src/scripts/python/TMB.py

```diff
@@ -0,0 +1,119 @@
+
+vcf = open(snakemake.input.vcf)
+artifacts = open(snakemake.input.artifacts)
+output_tmb = open(snakemake.putput.tmb, "w")
+
+
+FFPE_SNV_artifacts = {}
+header = True
+for line in artifacts:
+    if header:
+        header = False
+        continue
+    lline = line.strip().split("\t")
+    chrom = lline[0]
+    pos = lline[1]
+    key = chrom + "_" + pos
+    type = lline[2]
+    if type != "SNV":
+        continue
+    observations = int(lline[3])
+    FFPE_SNV_artifacts[key] = observations
+
+
+nr_TMB = 0
+header = True
+prev_pos = ""
+prev_chrom = ""
+for line in vcf:
+    if header:
+        if line[:6] == "#CHROM":
+            header = False
+        continue
+    lline = line.strip().split("\t")
```

Comment:
maybe skip first line by reading it before the loop
```
next(vcf)
for line in artifacts:
    lline = line.strip().split("\t")
```

Reply:
Will not work as there are several header lines.

Reply:
true